cypress/integration/frontpageLayoutSpec.js

```diff
@@ -23,7 +23,7 @@ Object.keys(services).forEach(index => {
           cy.get(el.header)
             .should('have.lengthOf', 1)
             .find('a')
-            .should('have.attr', 'href', 'https://www.bbc.co.uk/news') // expect `${config.baseUrl}${serviceConfig.url}` once header hooked up
+            .should('have.attr', 'href', `${serviceConfig.url}`)
```

Comment:
Using `${serviceConfig.url}` in favour of ` ${config.baseUrl}${serviceConfig.url}` since BrandContainer passes a relative url